tests/test_examples_app.py

```diff
@@ -41,7 +41,11 @@ def example_app():
     yield webapp
 
     # Stop server
-    os.killpg(webapp.pid, signal.SIGTERM)
+    try:
```

Comment:
In case you run into troubles in the future, it's ok to remove the entire example app. See:

https://github.com/inveniosoftware/rfcs/blob/master/rfcs/framework-0021-example-apps-removal.md